@vates/disposable/USAGE.md

```diff
@@ -0,0 +1,14 @@
+```js
```

Comment:
Add a link to https://github.com/JsCommunity/promise-toolbox#resource-management